integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/strategy/outbox/OutboxPollingOutOfOrderIdsIT.java

```diff
@@ -333,16 +337,20 @@ public void processDifferentRoutesUpdates_outOfOrder() {
 			);
 		} );
 
+		// Compared with the in-order test we have just this change SECOND <--> FIRST.
+		// That seems to be reasonable.
```

Comment:
It seems the order is not predictable here (judging from your use of `createdThenExecutedOutOfOrder()`), so I think this comment is obsolete?